inference-engine/src/transformations/src/transformations/common_optimizations/common_optimizations.cpp

```diff
@@ -53,6 +54,9 @@ bool ngraph::pass::CommonOptimizations::run_on_function(std::shared_ptr<ngraph::
     manager.register_pass<ngraph::pass::HSwishFusion>();
     manager.register_pass<ngraph::pass::ConvertPadToGroupConvolution>();
     manager.register_pass<ngraph::pass::NormalizeL2Fusion>();
+    manager.register_pass<ngraph::pass::BidirectionalLSTMSequenceDecomposition>();
+    manager.register_pass<ngraph::pass::BidirectionalRNNSequenceDecomposition>();
+    manager.register_pass<ngraph::pass::BidirectionalGRUSequenceDecomposition>();
```

Comment:
Do we need to do those decompositions for all plugins?

Reply:
I will clarify this point.

Reply:
cpu/gpu/vpu/onednn plugins don't support Bidirectional mode yet. 
As far as I know, there are no plans to add such support anytime soon.
Anyway, the transformations can be disabled using callbacks.